.github/workflows/compile-with-typescript-v4.yml

```diff
@@ -22,9 +22,9 @@ jobs:
           node-version: 16
           cache: "pnpm"
       - name: Remove packages that can't be compiled with TypeScript v4
-        run: rm -fr packages/hardhat-viem packages/hardhat-toolbox-viem packages/hardhat-web3-v4
+        run: rm -fr packages/hardhat-viem packages/hardhat-toolbox-viem packages/hardhat-toolbox packages/hardhat-web3-v4
```

Comment:
We have decided to ignore this check for hardhat-toolbox as well as the previously excluded hardhat-toolbox-viem.

Reply:
Ignition is compiled against typescript v5, this would blow up in this check.
As this is a major version change to the toolbox, we think this is acceptable.